installation.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Installation manual"
+header-includes:
+    <link rel="Metafacture Icon" type="image/x-icon" href="/img/metafacture-icon.png">
+---
+
+## Getting started
+
```

Comment:
This page is a copy of the README of metafacture-core. Question is if we could have _one_ place to curate the content as the content may change (see e.g. https://github.com/metafacture/metafacture-core/pull/417/commits/8bb0808d9974546aeb59593a4061bef2d6a97a73).
 We may a) just link to the section or b) copy the section automatically if it changes

Reply:
Since this is not a whole copy of [metafacture-core README](https://github.com/metafacture/metafacture-core/blob/master/README.md) but starts at the ["Getting started"-section](https://github.com/metafacture/metafacture-core/blob/master/README.md#getting-started) and in line 1 - 5 contains additional meta informations for generating the html document, I do not think that there's a way to copy or link the file.
Another possibility is to change the structure of the website and the md-files in the repo to fit to each other, so it's possible to copy or to link.

Reply:
@TobiasNx what do you think? I think it was you who wished to have an installation guide on the page.

Reply:
I think a getting started / installation site should be on metafacture.org without linking. But since the tool for the website is going to be change I am okay with just linking.